index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./src/styles.css">
+	<title>Todo List</title>
+</head>
+
+<body>
+	<header>
+		<div class="title-date">
+			<h1>Todo List</h1>
+			<p id="date">date</p> <!--this will display the date-->
```

Comment:
There is a tag for that: https://www.w3schools.com/tags/tag_time.asp

I think the `id` it's explicit enough you don't need that comment

Reply:
👌